gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -108,18 +113,45 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
 
     mapContainerViewModel = getViewModel(MapContainerViewModel.class);
     projectSelectorViewModel = getViewModel(ProjectSelectorViewModel.class);
+    featureSelectorViewModel = getViewModel(FeatureSelectorViewModel.class);
+
+    featureSelectorDialogFragment = new FeatureSelectorFragment(featureSelectorViewModel);
 
     viewModel = getViewModel(HomeScreenViewModel.class);
     viewModel.getProjectLoadingState().observe(this, this::onActiveProjectChange);
     viewModel
         .getShowAddFeatureDialogRequests()
         .observe(this, e -> e.ifUnhandled(this::onShowAddFeatureDialogRequest));
     viewModel.getBottomSheetState().observe(this, this::onBottomSheetStateChange);
+    viewModel
+        .getOverlappingFeatures()
```

Comment:
Naming nit: From the name it's not clear when this observable emits items; interpreted naively it would imply it always returns all overlapping features. Is there a more self-describing name we could use here?

Reply:
I switched this to `LiveData` and retained the name, since it mirrors our other naming conventions.